### ISSUE TITLE
Rendering routed panel with mousebites

### ISSUE THREAD

**Issue by ju5t** [tracespace/tracespace#84]
Unfortunately I can't share the gerber files (or final renders) in public. 

It's a square panel with 4 round PCBs and 4 small square PCBs, routed around the edges and connected with mousebites. Mousebites are pretty common for us.

An example of the outline from the tracespace viewer:

![screen shot 2018-08-07 at 10 57 55](https://user-images.githubusercontent.com/3635751/43765670-c986c15c-9a30-11e8-92d1-f07a89ab9fc0.png)

We're trying to use pcb-stackup to create a top and bottom view. 

- maskWithOutline == false, blue in the screenshot becomes black, instead of the soldermask.
- maskWithOutline == true, it cuts out the PCBs completely.

The next part might be a separate issue.

The border of the PCBs has a gold finish (along with the pads, etc.) but for some reason the border  isn't covering the entire circle on the 2nd and 4th PCB in the panel. It's covering 90-360 degrees of the PCB, the first 90 degrees isn't there.

I can replicate this on the viewer. I can send the files via e-mail and I'm happy to provide more feedback if needed.

**Comment by kasbah** [comment]
Hey, I am a bit confused about the image you posted. I don't see any tabs/mousebites and can't tell what's PCB and what is cut-out. 

If you send me the files privately via email or gitter I can take a look though I can't promise I'll be able to solve it of course.

**Comment by ju5t** [comment]
It turns out something is off the outline file that we have. The screenshot in the initial post shows the outline in the tracespace viewer. I haven't been able to replicate this yet in gerbv or gerbview, they both show the outline as normal.

`$ gerbv -o test.png -x png contour.gko` 

results in:

![test](https://user-images.githubusercontent.com/3635751/43782174-04b80ab4-9a5f-11e8-9b7f-0f4851578f4f.png)

I haven't received any feedback yet on which software was used to generate the outline.

**Comment by kasbah** [comment]
I believe tracespace viewer tries to render the outline layer as a filled in contiguous shape so that's why it looks different.

**Comment by ju5t** [comment]
I got some feedback and it comes down to 'why on earth are you using these old files'. 

I converted `contour.gko` (import/export in another program) and it now renders properly in the tracespace viewer. For some reason it didn't occur to me today that I could try this with all files too, as we're still missing 1/4th of finish in PCB two and four in the mentioned panel. I'll try that tomorrow and give those files a go.
